.github/workflows/submodule-sync.yml

```diff
@@ -0,0 +1,21 @@
+name: Submodule Sync
```

Comment:
Is this required for seats and slots?

Reply:
It was one of the commits that was pulled from main. Should I delete this file since it's probably already there?